library/blivet.py

```diff
@@ -1131,32 +1131,43 @@ def _apply_defaults(self):
                 if self._pool.get(name) is None:
                     self._pool[name] = default
 
+    def _create_one_member(self, disk):
+        if not disk.isleaf or disk.format.type is not None:
+            if safe_mode and disk.format.name != get_format(None).name:
+                raise BlivetAnsibleError(
+                    "cannot remove existing formatting (%s) and/or devices on disk '%s' (pool '%s') in safe mode" %
+                    (disk.format.type, disk.name, self._pool['name'])
+                )
+            else:
+                self._blivet.devicetree.recursive_remove(disk)
+
+        if use_partitions:
+            label = get_format("disklabel", device=disk.path)
+            self._blivet.format_device(disk, label)
+            member = self._blivet.new_partition(parents=[disk], size=Size("256MiB"), grow=True)
```

Comment:
why 256MiB?

Reply:
That would be a question for @dwlehman, I just moved the code from `_create_members` to be able to reuse it when creating only a single new member. I assume this is just a magic minimal size for members. In theory minimal size for LVM PV is 8 MiB, but that wouldn't be really useful.